Users/abehenderson/CSX/Closures/defineFirstArg.js

```diff
@@ -0,0 +1,32 @@
+// Attempt #1
+
+// Create a function called `defineFirstArg` that accepts a function called `callback` and an argument
+function defineFirstArg(callback, num) {
+  // `defineFirstArg` will return a function called `newFunction`
+  function newFunction(...args) {
```

Comment:
instead of returning below, you can simply return the function here like this: 
`return function newFunction(...args)`